dev/com.ibm.ws.org.eclipse.yasson.1.0/bnd.overrides

```diff
@@ -48,5 +50,5 @@ Service-Component:\
   com.ibm.ws.org.eclipse.yasson.ResourceProvider;\
     implementation:=com.ibm.wsspi.classloading.ResourceProvider;\
     provide:=com.ibm.wsspi.classloading.ResourceProvider;\
-    configuration-policy:=ignore;\
+    configuration-policy:=require;\
```

Comment:
The creative (ab)use of metatype and internal configuration is effective for this, but we should think about if there is a more proper way of implementing.  Approving for now because it solves the problem.